lib/collection/src/collection/shard_transfer.rs

```diff
@@ -279,6 +279,8 @@ impl Collection {
     ) -> CollectionResult<()> {
         // TODO: Ensure cancel safety!
 
+        log::debug!("Starting abort shard transfer {transfer_key:?}");
+
```

Comment:
Aborting is not a long running process, and so I'd prefer to not log a separate start/stop, but to only log the event once.

I think it's best to place the message right before the bigger if-block on line 308.

Reply:
It's not just to track time but also if we get any error in between. 

In case of error, only start will be printed but not stop. So let's keep this. Wdyt?